modules/op-running-pipelines-in-restricted-environment.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// pipelines/creating-applications-with-cicd-pipelines
+
+[id="op-running-pipelines-in-restricted-environment_{context}"]
+=  Running pipelines in a restricted environment
+
+
+To run OpenShift Pipelines in a disconnected cluster or a cluster provisioned in a restricted environment, you must ensure that either the Samples Operator is configured for a restricted network, or a cluster administrator has created a cluster with a mirrored registry. To learn more about Samples Operator and mirrored registries, see the Additional resources section.
+
+The following section uses the `Pipelines Tutorial` example to create a pipeline for an application in a restricted environment using a cluster with a mirrored registry.
+
+.Procedure
+To ensure that the front-end interface, `vote-ui`, and the back-end interface, `vote-api`, of the `pipelines-tutorial` example work in a restricted environment, you must mirror the respective builder images from the mirror registry.
+
+. For the front-end interface, `vote-ui`, verify that the required images tag is not imported:
++
+[source,terminal]
+----
+$ oc describe imagestream python -n openshift
+----
++
+.Example output
+[source,terminal]
+----
+Name:			python
+Namespace:		openshift
+[...]
+
+3.8-ubi8 (latest)
+  tagged from registry.redhat.io/ubi8/python-38:latest
+    prefer registry pullthrough when referencing this tag
+
+  Build and run Python 3.8 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-python-container/blob/master/3.8/README.md.
+  Tags: builder, python
+  Supports: python:3.8, python
+  Example Repo: https://github.com/sclorg/django-ex.git
+
+  ! error: Import failed (InternalError): Internal error occurred: registry.redhat.io/ubi8/python-38:latest: Get "https://registry.redhat.io/v2/": Forbidden
```

Comment:
I would omit this line, it's not important for this doc and might scary the reader.